src/core_plugins/tagcloud/public/tag_cloud.js

```diff
@@ -397,23 +396,13 @@ function getFill(tag) {
   return colorScale(tag.text);
 }
 
-/**
- * Hash a string to a number. Ensures there is no random element in positioning strings
- * Retrieved from http://stackoverflow.com/questions/26057572/string-to-unique-hash-in-javascript-jquery
- * @param string
- */
-function hashCode(string) {
-  string = JSON.stringify(string);
+function hashWithinRange(str, max) {
+  str = JSON.stringify(str);
```

Comment:
Isn't tag.text always a string? I'm not sure why this line is necessary.

Reply:
yes, it's always a string.